### PR TITLE
Added save_job method for push job object in etcd

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Review by shtripat — COMMENTED** [review]
LGTM

File: tendrl/ceph_integration/central_store/__init__.py [hunk 1]
```diff
@@ -44,3 +44,6 @@ def save_ecprofile(self, ec_profile):
 
     def save_utilization(self, utilization):
         tendrl_ns.etcd_orm.save(utilization)
+
+    def save_job(self, job):
+        tendrl_ns.etcd_orm.save(job)
```
Comment (r0h4n) [inline tendrl/ceph_integration/central_store/__init__.py:49]: Why is this required?

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/central_store/__init__.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/ceph_integration/central_store/__init__.py:49]: @r0h4n without this function ceph integration can't create or update jobs, job class is in common but it is stored  using save_job function in all components.

Comment (GowthamShanmugam) [reply tendrl/ceph_integration/central_store/__init__.py:49]: @r0h4n ah i forgot if i add in common then it is not required in all component..

**Pull request closed without merging**